ci/environment-py37-upstream-master.yml

```diff
@@ -1,7 +1,7 @@
 name: test_env_xgcm
 dependencies:
-  - python=3.5
-  - xarray
+  - python=3.7
+  - git+https://github.com/pydata/xarray.git
```

Comment:
This does not work here. It needs to be in the pip section. See https://github.com/MITgcm/xmitgcm/blob/master/ci/environment-py36-xarraymaster.yml for a working example.

That's why the build failed: https://travis-ci.org/xgcm/xgcm/jobs/592737894